src/core/dbus-execute.c

```diff
@@ -2808,6 +2843,56 @@ int bus_exec_context_set_transient_property(
                         return 1;
                 }
 
+        } else if (streq(name, "MountImages")) {
+                char *source, *destination;
+                int permissive;
+                bool empty = true;
+
+                r = sd_bus_message_enter_container(message, 'a', "(ssb)");
+                if (r < 0)
+                        return r;
+
+                while ((r = sd_bus_message_read(message, "(ssb)", &source, &destination, &permissive)) > 0) {
+                        if (!path_is_absolute(source))
+                                return sd_bus_error_setf(error, SD_BUS_ERROR_INVALID_ARGS, "Source path %s is not absolute.", source);
```

Comment:
should insist on path_is_normalized() too I think. At the minimum for the mount destination

Reply:
ok, done for both - unit file parsing already did it as well